addons/payment/models/payment_method.py

```diff
@@ -295,6 +299,25 @@ def _get_compatible_payment_methods(
                 reason=REPORT_REASONS_MAPPING['express_checkout_not_supported'],
             )
 
+        # Filter pms that does not support manual capture.
+        unfiltered_pms = payment_methods
+        non_supported_manual_capture_pms = payment_methods.filtered(
+            lambda p: not p.support_manual_capture
+        )
+        providers = self.env['payment.provider'].browse(provider_ids)
+        capture_manually_supported_pp = providers.filtered('capture_manually')
```

Comment:
This name is misleading as this is not a list of providers that support manual capture, but rather a list of providers that have manual capture turned on.